src/httpfetch.cpp

```diff
@@ -276,6 +276,9 @@ HTTPFetchOngoing::HTTPFetchOngoing(const HTTPFetchRequest &request_,
 	curl_easy_setopt(curl, CURLOPT_CONNECTTIMEOUT_MS,
 			request.connect_timeout);
 
+	// Use HTTP/2 if possible
+	curl_easy_setopt(curl, CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_NONE);
```

Comment:
pretty sure this is the default as long as you're using https

Reply:
I've tested this by watching my server logs - before it was HTTP/1.1 and now it's HTTP/2

Reply:
tbh this may be something for another PR anyway

Reply:
https://curl.se/libcurl/c/CURLOPT_HTTP_VERSION.html:
> Default
> Since curl [7.62.0](https://curl.se/changes.html#7_62_0): CURL_HTTP_VERSION_2TLS
> Before that: CURL_HTTP_VERSION_1_1

IMO no need to change this if libcurl > 2018 does it anyway

Reply:
ahhh I guess it's just clients with pre-7.62

Reply:
ok, removed